test/functional/unified-spec-runner/runner.ts

```diff
@@ -143,11 +144,8 @@ export async function runUnifiedTest(
     // test runners MUST execute a non-transactional distinct command on
     // each mongos server before running any test that might execute distinct within a transaction.
     // To ease the implementation, test runners MAY execute distinct before every test.
-    if (
-      ctx.topologyType === uni.TopologyType.sharded ||
-      ctx.topologyType === uni.TopologyType.shardedReplicaset ||
-      ctx.topologyType === uni.TopologyType.loadBalanced
-    ) {
+    const topologyType = ctx.configuration.topologyType;
+    if (topologyType === TopologyType.Sharded || topologyType === TopologyType.LoadBalanced) {
```

Comment:
Just curious, what are the uni.TopologyType types for? Also, is there a follow up ticket already created for the second half of the description?